services/management/ia/templates_service_test.go

```diff
@@ -81,3 +86,38 @@ func TestCollect(t *testing.T) {
 		assert.Contains(t, rules, "user2_rule")
 	})
 }
+
+func TestConvertTemplate(t *testing.T) {
```

Comment:
fails with the error `pq: duplicate key value violates unique constraint "pg_database_datname_index"` if I use a `t.Parallel here`